### DISCUSSION TITLE
Reduce a titlebar height to make it more like it looks in iTerm2.

### DISCUSSION THREAD

**Discussion by hostmaster** [ghostty-org/ghostty#3735] [Feature Requests, Ideas] [answered]
OS: MacOS. This is quite subjective. In my opinion, a title bar with tabs looks better in iTerm2. Setting `macos-titlebar-style = tabs` with a lower height would likely enhance its appearance.
![2024-12-28 16 30 01](https://github.com/user-attachments/assets/7b5ecc1d-9bd9-49da-876b-7a5e237543d3)

**Comment by andyl** [comment]
I would also like to reduce the height of the titlebar and tab-bar on Linux (Xubuntu 22.04)

**Comment by JakeWharton** [comment, marked as answer]
Duplicate of #3297
